contributors/devel/running-locally.md

```diff
@@ -121,10 +121,10 @@ Note the difference between a [container](https://kubernetes.io/docs/user-guide/
 and a [pod](https://kubernetes.io/docs/user-guide/pods/). Since you only asked for the former, Kubernetes will create a wrapper pod for you.
 However you cannot view the nginx start page on localhost. To verify that nginx is running you need to run `curl` within the docker container (try `docker exec`).
 
-You can control the specifications of a pod via a user defined manifest, and reach nginx through your browser on the port specified therein:
+You can control the specifications of a pod via a user defined manifest, and reach nginx through your browser on the port specified there in:
 
 ```sh
-cluster/kubectl.sh create -f test/fixtures/doc-yaml/user-guide/pod.yaml
+cluster/kubectl.sh create -f test/fixtures/doc-yaml/user-guide/multi-pod.yaml
```

Comment:
`git log -- test/fixtures/doc-yaml/user-guide/pod.yaml` shows that this was removed in https://github.com/kubernetes/kubernetes/commit/2ae916a0ba0363c0381699c2c32b686180fbda08 (https://github.com/kubernetes/kubernetes/pull/54342) as it was thought as not used anywhere.

IMO we should it add it back again instead of changing this to `multi-pod`.yaml. Created https://github.com/kubernetes/kubernetes/pull/65962 to add it back. Ptal.

Reply:
Thanks for your response, Nikhita!  I tried ```deployment.yaml``` and ```multi-pod.yaml```, they both worked well.  And the guide is to use a pod as an example so I thought maybe the easiest way was to just change the guideline. But yes, I agree that it's better to add the ```pod.yaml``` back :)

Reply:
Closed this PR.